### PR TITLE
fix: flaky cache test on macOS

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
tests, fix

**Did you add tests for your changes?**
yes
**If relevant, did you update the documentation?**
no
**Summary**
I see some flakiness in the cache test on macOS after the release of `5.10.0`.

References - 
https://github.com/webpack/webpack-cli/runs/1503646276?check_suite_focus=true
https://github.com/webpack/webpack-cli/runs/1503607410?check_suite_focus=true

**Does this PR introduce a breaking change?**
no

**Other information**
no

### CODE REVIEW

**Review by alexander-akait — CHANGES_REQUESTED** [review]
Bad idea, we should search way why it happens, not break our tests

**Comment by snitin315** [comment]
from CI -
Screenshot at 2020-12-05 20-30-24

I will investigate 👍🏻

**Comment by alexander-akait** [comment]
Do you run test locally?

**Comment by snitin315** [comment]
yeah, passing locally 😕

**Comment by alexander-akait** [comment]
It means other tests generate cache too, so we got duplicate

File: test/cache/cache.test.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ describe('cache', () => {
     });
 
     it('should work in multi compiler mode', () => {
-        rimraf.sync(path.join(__dirname, '../../node_modules/.cache/webpack/cache-test-{first,second}-development'));
+        rimraf.sync(path.join(__dirname, '../../node_modules/.cache/webpack/**'));
```
Comment (alexander-akait) [inline test/cache/cache.test.js:38]: It is unsafe, don't remove full cache, it breaks other tests, in test you need delete only necessary caches

Comment (snitin315) [reply test/cache/cache.test.js:38]: How about defining a separate cache-directory for separate tests 😄

Comment (alexander-akait) [reply test/cache/cache.test.js:38]: Can you clarify?

Comment (snitin315) [reply test/cache/cache.test.js:38]: `node_modules/.cache/webpack` is the default directory for the cache we can overwrite with `cacheDirectory` option in config.

Comment (alexander-akait) [reply test/cache/cache.test.js:38]: I am already do it on tests level

**Comment by alexander-akait** [comment]
Thanks for the PR, fixed in master, temporary disable check on `2`, I will investigate with the problem in the future

**Pull request closed without merging**